src/Utilities/matrix_of_constraints.jl

```diff
@@ -316,6 +316,18 @@ function _add_set(sets, i, func::MOI.AbstractVectorFunction)
     return add_set(sets, i, MOI.output_dimension(func))
 end
 
+const _MODIF_NOT_ALLOWED = """
```

Comment:
I've been trying to remove all abbreviations from the source. Can we make this `_MATRIXOFCONSTRAINTS_MODIFY_NOT_ALLOWED_ERROR_MESSAGE`